modules/cert-manager-self-signed-about.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-self-signed.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-self-signed-about_{context}"]
+= About self-signed issuers
+
+TODO
+
+// TODO: Need to learn more about what the use case really is for this - just for boostrapping a CA issuer?
```

Comment:
@thejasn @swghosh I had this question in the gdoc, but I'm looking for details on what the use case is here. Is bootstrapping a CA issuer really the main/only use case we need to cover for this?

The text there mentioned that self-signed issuers weren't really for production, mostly for testing. Is that true even for bootstrapping a CA issuer?

Reply:
Closing comment for now since we're not doing self-signed for GA